dipy/reconst/dki.py

```diff
@@ -359,6 +361,75 @@ def _F2m(a, b, c):
     return F2
 
 
+def _directional_kurtosis(dt, MD, kt, V, min_diffusivity=0, min_kurtosis=-1):
```

Comment:
Is it real necessary to convert _direction_kurtosis to a public function? Users can just use the more generalized function version apparent_kurtosis_coef which deals with both single and multi-voxel data.